data/card_catalogue/food.json

```diff
@@ -18,7 +18,7 @@
     "tier": "green",
     "details": {
       "rules": {
-        "active": "Summon a brick of Ambrosia. Any NPC who has tasted or smelt it before will find it irresistible.",
+        "active": "Summon an irresistable brick of Ambrosia, food of the Gods.",
```

Comment:
Changing the wording here to be less of a mouthful. I think this is enough info for the players to make an interpretation